src/components/Paginate.js

```diff
@@ -85,7 +85,7 @@ const Paginate = ({ count, active, urlF, onClick }) => {
   );
 
   return (
-    <Pagination aria-label="Table page" className="mt-3">
+    <Pagination aria-label="Table page" className="mt-3 pagination">
```

Comment:
My previous comment (edit) still stands in that manually adding the class is completely redundant. 
```suggestion
    <Pagination aria-label="Table page" className="mt-3">
```